Error: Unsupported attribute
│
│   on outputs.tf line 2, in output "data_ingress_table_arn":
│    2:     value = module.platform_dynamodb_table["DataIngress"].arn
│     ├────────────────
│     │ module.platform_dynamodb_table["DataIngress"] is a object, known only after apply
│
│ This object does not have an attribute named "arn".
╵
time=2023-09-21T17:22:40+05:30 level=error msg=Terraform invocation failed in C:/.terragrunt-cache/XTQi-oi0p63BODQSy4EBJhN-Zak/VlJQvwBldSrlCuJoozEeVg97DMM/platform-datastore prefix=[C:\Users\umababy.kannan\OneDrive - HCL Technologies Ltd\Documents\Star CAP\platform-infra\live\dev\eu-central-1\platform-datastore]

time=2023-09-21T17:22:40+05:30 level=error msg=1 error occurred:
        * exit status 1
